test_login.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+base_url = 'https://python-qa-training.herokuapp.com/api/auth/login'
+email = 'sievdokymov@lohika.com'
+password = '12345678'
+
+def login(email,password):
+    res = requests.post(base_url, json={"email":email, "password":password})
+    return res
+
+def test_login_valid_credentials():
+    assert login(email,password).status_code == 200
+
+def test_login_invalid_email():
+    email = "wrong@wrong.com"
+    assert login(email,password).status_code == 404
+
+def test_login_invalid_password():
+        password = "wrong"
+        assert login(email, password).status_code == 401 #unathorized
+
+def test_login_empty_values():
+    email = ""
```

Comment:
Since email and password are global attributes, the common practice would call `global` keyword to avoid variable shadowing.

More on variable shadowing - https://en.wikipedia.org/wiki/Variable_shadowing

```
global email
email = ""
global password
password = ""
```
The code above would be correct, however, it makes code to be quite huge for such an easy case. As you already have `login() function with `email` and `password` arguments, to keep one-liner code style, you may consider using such construction:
`login(email="", password="") ` which does not use global attributes at all but does the same logic.

Reply:
@bochkala appropriate part of the code was removed due to made refactorings - pls take a look at recently submitted "homework1-5" pull request